### PR TITLE
X Rebrand strings

### PR DESCRIPTION
**Pull request by lucasnantonio**
Replacing most mentions of Tweet, Twitter, Retweets, Quote Tweets etc to the new language.
We're not updating the python code files or "under the hood" tables for now.

### CODE REVIEW

**Comment by jonahgrant** [comment]
lgtm

File: README.md [hunk 1]
```diff
@@ -8,13 +8,13 @@ This repository is a place for us to transparently host our content, algorithms,
...
 
-The folder `/documentation` holds the [Markdown content](https://github.com/twitter/communitynotes/tree/main/documentation) that is used to generate our [documentation website](https://communitynotes.twitter.com/guide).
+The folder `/documentation` holds the [Markdown content](https://github.com/twitter/communitynotes/tree/main/documentation) that is used to generate our [documentation website](https://communitynotes.x.com/guide).
 
 Here you can also find our [research paper](https://github.com/twitter/communitynotes/blob/main/birdwatch_paper_2022_10_27.pdf).
 
 ## About Community Notes
 
-Community Notes aims to create a better informed world, by empowering people on Twitter to add helpful notes to Tweets that might be misleading.
+Community Notes aims to create a better informed world, by empowering people on X to add helpful notes to posts that might be misleading.
```
Comment (lavingiasa) [inline README.md:17]: I believe these are called "x's" now rather than posts: https://twitter.com/elonmusk/status/1683365872323133442?s=20

Comment (lucasnantonio) [reply README.md:17]: That is 4 days old! Things here move too fast now 😅

**Comment by lavingiasa** [comment]
Good point! Feel free to close. Appreciate the reply.

**Pull request merged**